napari/layers/surface/surface.py

```diff
@@ -150,9 +151,11 @@ def __init__(
         # assign mesh data and establish default behavior
         if len(data) not in (2, 3):
             raise ValueError(
-                'Surface data tuple must be 2 or 3, specifying'
-                'verictes, faces, and optionally vertex values,'
-                f'instead got length {len(data)}.'
+                trans._(
+                    'Surface data tuple must be 2 or 3, specifying'
+                    'vertices, faces, and optionally vertex values,'
+                    'instead got length {}.'.format(len(data))
+                )
```

Comment:
This should be

```python
trans._(
    'Surface data tuple must be 2 or 3, specifying'
    'vertices, faces, and optionally vertex values,'
    'instead got length {length}.'
    deferred=True,
    length=len(data),
)
```

More examples are available here @DragaDoncila 

https://napari.org/guides/stable/translations.html#deferred-translations

We use now deferred translations when raising errors :)